### ISSUE TITLE
Huge difference between Stage 1 accuracy and Stage 2 accuracy

### ISSUE THREAD

**Issue by dreamstorm25** [michalfaber/tensorflow_Realtime_Multi-Person_Pose_Estimation#8]
I am training a model based on your repository but it does pose estimation on 13 keypoints. The code has been modified to adapt to new configuration. During training I noticed that there is a huge difference between the L1_acc and L2_acc of any stage. While L2_acc has gone over 0.99 and is steadily increasing, L1_acc is unstable as it increases and decreases without any pattern.  L1_acc is about the PAFs and L2_acc is for confidence maps. Could you help me figure out what the issues can occur while predicting PAFs ?
![diff](https://user-images.githubusercontent.com/22818339/76687326-d8ad1680-65f0-11ea-9173-d532ea570c77.png)

**Comment by Bardhitoo** [comment]
Having the same issue - did you solve it?

**Comment by dreamstorm25** [comment]
I have noticed that my results are fine(not always spot on but mostly good), so I am guessing accuracy doesn't play that much of a part in PAFs, it's the loss you should be looking at.

**Comment by kuanyshbakytuly** [comment]
> I am training a model based on your repository but it does pose estimation on 13 keypoints. The code has been modified to adapt to new configuration. During training I noticed that there is a huge difference between the L1_acc and L2_acc of any stage. While L2_acc has gone over 0.99 and is steadily increasing, L1_acc is unstable as it increases and decreases without any pattern. L1_acc is about the PAFs and L2_acc is for confidence maps. Could you help me figure out what the issues can occur while predicting PAFs ? ![diff](https://user-images.githubusercontent.com/22818339/76687326-d8ad1680-65f0-11ea-9173-d532ea570c77.png)

Could you share the code for evaluating accuracy?
